packages/jest-mock/src/index.js

```diff
@@ -109,15 +122,15 @@ function isReadonlyProp(object, prop) {
   );
 }
 
-function getSlots(object) {
+function getSlots(object?: Object): Array<string> {
   const slots = {};
   if (!object) {
     return [];
   }
 
   let parent = Object.getPrototypeOf(object);
   do {
-    if (object === Function.prototype) {
+    if (object === Object.getPrototypeOf(Function)) {
       break;
```

Comment:
This is that case we talked about @cpojer - where Flow was throwing on the prototype not being defined in the statics of `Function`.